modules/virt-virtctl-commands.adoc

```diff
@@ -29,70 +29,200 @@ To view a list of global command options that you can use with any `virtctl` com
+
+.`virtctl console, expose, vnc` commands
+[width="100%",cols="1a,2a",options="header"]
+|===
+|Command |Description
 
 |`virtctl console <vmi_name>`
-|Connect to a serial console of a virtual machine instance.
+|Connect to the serial console of a VMI.
+
+|`virtctl expose <vm_name>`
+|Create a service that forwards a designated port of a VM or VMI and expose the service on the specified port of the node.
 
 |`virtctl vnc --kubeconfig=$KUBECONFIG <vmi_name>`
-|Open a VNC (Virtual Network Client) connection to a virtual machine instance. Access the graphical console of a virtual machine instance through a VNC which requires a remote viewer on your local machine.
+|Open a Virtual Network Client (VNC) connection to a VMI.
+
+Accessing the graphical console of a VMI through VNC requires a remote viewer on your local machine.
+
+|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only=true <vmi_name>`
+|Display the port number and connect manually to a VMI by using any viewer through the VNC connection.
+
+|`virtctl vnc --kubeconfig=$KUBECONFIG --port=<port-number> <vmi_name>`
+|Specify a port number to run the proxy on the specified port, if that port is available.
+
+If a port number is not specified, the proxy runs on a random port.
+|===
+
+[id='exporting-vms_{context}']
+== Exporting VM volumes
+
+You can use `virtctl vmexport` commands to create, download, or delete a volume exported from a VM, VM snapshot, or persistent volume claim (PVC).
+
+.`virtctl vmexport` commands
+[width="100%",cols="1a,2a",options="header"]
+|===
+|Command |Description
+
+|`virtctl vmexport create <vmexport_name> --vm\|snapshot\|pvc=<object_name>`
+|Create a `VirtualMachineExport` custom resource (CR) to export a volume from a VM, VM snapshot, or PVC.
+
+* `--vm`: Exports the PVCs of a VM.
+* `--snapshot`: Exports the PVCs contained in a `VirtualMachineSnapshot` CR.
+* `--pvc`: Exports a PVC.
+* Optional: `--ttl=1h` specifies the time to live. The default duration is 2 hours.
+
+|`virtctl vmexport delete <vmexport_name>`
+|Delete a `VirtualMachineExport` CR manually.
+
+|`virtctl vmexport download <vmexport_name> --output=<output_file> --volume=<volume_name>`
+|Download the volume defined in a `VirtualMachineExport` CR.
+
+* `--output` specifies the file format. Example: `disk.img.gz`.
+* `--volume` specifies the volume to download. This flag is optional if only one volume is available.
+
+Optional:
+
+* `--keep-vme` retains the `VirtualMachineExport` CR after download. The default behavior is to delete the `VirtualMachineExport` CR after download.
+* `--insecure` enables an insecure HTTP connection.
+
+|`virtctl vmexport download <vmexport_name> --<vm\|snapshot\|pvc>=<object_name> --output=<output_file> --volume=<volume_name>`
+|Create a `VirtualMachineExport` CR and then download the volume defined in the CR.
+|===
+
+[id='vm-memory-dump_{context}']
+== VM memory dump
```

Comment:
This section seems to be a mix of a procedure and reference material. I think it would be best to take lines 135 to 164 and refactor them into a proper procedure and pull the table (lines 166 to 191) into a separate reference module. I think that following the spirit of the mod docs rules, the reference content above should also be modularized. That issue is less urgent, since the content above is all reference material.